core/src/main/scala/org/http4s/JsonDecoder.scala

```diff
@@ -0,0 +1,13 @@
+package org.http4s
+
+trait JsonDecoder[F[_], A] {
```

Comment:
This trait doesn't have anything to do with JSON.  This is more of a typeclass for decoders without any HTTP knowledge.